docs/contributing-analyzer.md

```diff
@@ -17,28 +17,34 @@ In general, it is best to run commands from the Visual Studio Developer Command
-    - **JAVA_HOME**
-    - **MSBUILD_PATH** - path to the MSBuild.exe executable (MSBuild 16)
+    - **JAVA_HOME** (e.g. `C:\Program Files\Java\jdk-11.0.2`)
+    - **MSBUILD_PATH** - path to the MSBuild.exe executable (MSBuild 16 e.g. `C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin\MSBuild.exe`)
     - **NUGET_PATH** - path to the nuget.exe executable (related to the [plugin integration tests](./contributing-plugin.md#integration-tests))
-    - **ORCHESTRATOR_CONFIG_URL** - url to orchestrator.properties file (for integration tests) in uri form (i.e. file:///c:/something/orchestrator.properties)
-    - **RULE_API_PATH** - path to folder containing the rule api jar
-    - **PATH** - the system **PATH** variable must contain:
-        - the path to the dotnet core installation folder
-        - the path to the MSBuild bin folder
-        - the path to the visual studo installer folder (for vswhere.exe)
-        - the path to the nuget executable folder (e.g. C:\Program Files\nuget)
-        - the path to the JDK bin folder
-        - %M2_HOME%\bin
-        - the path to the SonarScanner for .NET folder and to the Scanner CLI
+    - **Sonarsource internal only** These two steps require access to SonarSource internal resources and are not possible for external contributers
+        - **ORCHESTRATOR_CONFIG_URL** - url to orchestrator.properties file (for integration tests) in uri form (i.e. file:///c:/something/orchestrator.properties). See also: [Documentation in the orchestrator repository](https://github.com/sonarsource/orchestrator#configuration)
+        - **RULE_API_PATH** - path to folder containing the rule api jar. The rule api jar can be found at [repox.jfrog search](https://repox.jfrog.io/ui/artifactSearchResults?name=rule-api&type=artifacts) or [repox.jfrog private releases](https://repox.jfrog.io/ui/native/sonarsource-private-releases/com/sonarsource/rule-api/rule-api/)
+    - **PATH** - the **PATH** variable must contain (either *system* or *user* scope):
+        - [*System*] the path to the dotnet core installation folder (`C:\Program Files\dotnet\`)
+        - [*System*] the path to the MSBuild bin folder (`C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Current\Bin`)
+        - [*System*] the path to the visual studo installer folder (for vswhere.exe) (`C:\Program Files (x86)\Microsoft Visual Studio\Installer`)
+        - [*System*] the path to the nuget executable folder (`C:\Program Files\nuget`)
+        - [*System*] the path to the JDK bin folder (`C:\Program Files\Java\jdk-11.0.2\bin`)
+        - [*System*] %M2_HOME%\bin (`C:\Program Files\JetBrains\IntelliJ IDEA\plugins\maven\lib\maven3\bin` [Maven cli](https://maven.apache.org/install.html). Here installed via IntelliJ IDEA.)
+        - [*System*] the path to the SonarScanner for .NET folder and to the Scanner CLI ([SonarScanner download](https://github.com/SonarSource/sonar-scanner-msbuild/releases))
```

Comment:
here all the env vars are `system`, so maybe remove the prefix `[System]` and keep only a mention at the beginning - see like you did in line 131 ` * Add the following environment variables (*user* scope) `